src/main/MainTabsScreen.js

```diff
@@ -89,31 +89,48 @@ function MainTabsScreen(props: Props) {
           component={HomeScreen}
           options={{
             tabBarLabel: 'Home',
-            tabBarIcon: ({ color }) => <IconInbox size={24} color={color} />,
+            tabBarIcon: properties => (
+              <View style={styles.center}>
+                <IconInbox size={24} color={properties.color} />
+                {properties.focused && <Text style={styles.bottomNav}>Home</Text>}
+              </View>
+            ),
           }}
         />
         <Tab.Screen
           name="stream-tabs"
           component={StreamTabsScreen}
           options={{
             tabBarLabel: 'Streams',
-            tabBarIcon: ({ color }) => <IconStream size={24} color={color} />,
+            tabBarIcon: properties => (
```

Comment:
I think the `({ color })` destructuring syntax (that we're currently using) is a bit cleaner here, for these very short functions.

Reply:
But I had to check whether the component is active or not, fetched from the `focussed` key of the object, to render the label... 😕

Reply:
Right, so you can get that the same way: `({ color, focused }) => …`

Reply:
Oh sorry! I didn't read `destructuring syntax` 😅